### PR TITLE
TOAZ-297 followup: Use update MS Azure logo

### PR DESCRIPTION
**Pull request by rtitle**
Per Ilyana, this is the right logo to use on the Terra on Azure Preview screen.

image

### CODE REVIEW

File: src/pages/AzurePreview.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import { useState } from 'react'
 import { div, h, h1, p } from 'react-hyperscript-helpers'
 import { ButtonOutline, ButtonPrimary, Link } from 'src/components/common'
-import { ReactComponent as AzureLogo } from 'src/images/azure-new.svg'
+import { ReactComponent as AzureLogo } from 'src/images/ms-azure.svg'
```
Comment (cahrens) [inline src/pages/AzurePreview.js:4]: Can you delete the previous icon you checked in? And did you optimize the SVG (`yarn optimize-image-svgs`)?

Comment (rtitle) [reply src/pages/AzurePreview.js:4]: Yes, and no -- will do

Comment (rtitle) [reply src/pages/AzurePreview.js:4]: Done

**Review by cahrens — APPROVED** [review]

**Review by nawatts — APPROVED** [review]

**Comment by lstrano-broad** [comment]
Looks like the Terra & MS logo are not centered in the modal. Here is a SVG of the Terra/MS logo lockup. I also made the MS logo more proportional to Terra. Also can we underline the email link for accessibility? thx
terra-ms-logo-lockup

**Comment by rtitle** [comment]
Oh thanks @lstrano-broad  -- I'll update (was maybe a little quick to merge)

**Pull request merged**